src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -179,6 +191,14 @@ export class ExamNavigationBarComponent implements OnInit {
      * @return the sync status of the exercise (whether the corresponding submission is saved on the server or not)
      */
     setExerciseButtonStatus(exerciseIndex: number): 'synced' | 'synced active' | 'notSynced' {
+        // if we are in the exam timeline view use the following logic as not synced doesn't make sense there
+        this.icon = faCheck;
+        if (this.examTimeLineView && this.exerciseIndex === exerciseIndex) {
+            return 'synced active';
+        } else if (this.examTimeLineView) {
+            return 'synced';
+        }
```

Comment:
```suggestion
        if (this.examTimeLineView) {
            return this.exerciseIndex === exerciseIndex ? 'synced active' : 'synced';
        }
```
Nitpick, I think this makes it more clear that we differ from default behavior when in the timeline view.